### PR TITLE
Add install wizard tags to the client sdk docs

### PR DESCRIPTION
**Pull request by emmawillis**

### CODE REVIEW

File: docs/sdk/client-side-sdks/android/android-gettingstarted.md [hunk 1]
```diff
@@ -9,57 +9,60 @@ sidebar_custom_props: { icon: material-symbols:rocket }
 [![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/android-client-sdk)](https://search.maven.org/artifact/com.devcycle/android-client-sdk)
 [![GitHub](https://img.shields.io/github/stars/devcyclehq/android-client-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/DevCycleHQ/android-client-sdk)
 
+[//]: # (wizard-initialize-start)
```
Comment (ajwootto) [inline docs/sdk/client-side-sdks/android/android-gettingstarted.md:12]: hmm does this mean we're going with java over kotlin for android?

Comment (ajwootto) [reply docs/sdk/client-side-sdks/android/android-gettingstarted.md:12]: I think all new Android projects use Kotlin now, and its the default in their own docs I think?

Comment (ajwootto) [reply docs/sdk/client-side-sdks/android/android-gettingstarted.md:12]: actually looks like its more of a toss up in their docs, most of the time theres a choice of java or kotlin

Comment (emmawillis) [reply docs/sdk/client-side-sdks/android/android-gettingstarted.md:12]: hmm we could always just include both, if Cobi's cool with that

Comment (emmawillis) [reply docs/sdk/client-side-sdks/android/android-gettingstarted.md:12]: i guess in the meantime i'll switch it to kotlin

File: docs/sdk/client-side-sdks/javascript/javascript-openfeature.md [hunk 2]
```diff
@@ -56,11 +61,20 @@ await OpenFeature.setContext(user)
 await OpenFeature.setProviderAndWait(devcycleProvider)
 // Get the OpenFeature client
 const openFeatureClient = OpenFeature.getClient()
+```
+[//]: # (wizard-initialize-end)
+
 
+[//]: # (wizard-evaluate-start)
+
+Use a variable value by passing the variable key and default value to one of the OpenFeature flag evaluation methods
```
Comment (cdruxerman) [inline docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:70]: Variable should be capitalized

File: docs/sdk/client-side-sdks/javascript/javascript-usage.md [hunk 3]
```diff
@@ -41,6 +41,8 @@ devcycleClient.onClientInitialized((err) => {
 
 ## Using Variable Values
 
+[//]: # (wizard-evaluate-start)
+
 To get values from your Variables, `variableValue()` is used to fetch variable values using the identifier `key` coupled with a default value.
```
Comment (cdruxerman) [inline docs/sdk/client-side-sdks/javascript/javascript-usage.md:46]: Variable should be capitalized

**Review by cdruxerman — APPROVED** [review]

**Pull request merged**